docs/topics/launcher-review-update-booster-ci.adoc

```diff
@@ -21,6 +21,4 @@ $ git commit -m "Made an update"
 $ git push
 ----
 
-. Click _See it here!_ in _Creating your project on OpenShift Online_.
-
-Notice a new deployment is running! Once complete, click on the link for booster application at the top of the page and observe your changes appear.
\ No newline at end of file
+More details on interacting with your booster can be found here
```

Comment:
the link to the console is also on top of the page on the "While you wait..." section maybe it's better to remove it here
https://docs.google.com/drawings/d/15rMs1Pv69OjHZLXZRtdLTGLziIX3DuEMgrhM4kHjQOw/edit